src/cljs/reviews_next/pages/review.cljs

```diff
@@ -0,0 +1,81 @@
+(ns reviews-next.pages.review
+  (:require
+    [re-frame.core :as re-frame]
+    [cljs.core.async :refer [<!]]
+    [cljs-http.client :as http]
+    [clojure.pprint :as pp]
+    ["@material-ui/core" :as material-ui]
+    [stylefy.core :as stylefy :refer [use-style]]
+    [reviews-next.db :as db]
+    [reviews-next.events :as events]
+    [reviews-next.subs :as subs])
+  (:require-macros
+    [cljs.core.async.macros :refer [go]]))
+
+(def main-content-style
+  {:display "flex"
+   :flex-direction "row"})
+
+(defn section-style [width]
+  {:margin "40px"
+   :width width})
+(def box-button-style
+  {:display "flex"
+   :flex-direction "row"
+   :justify-content "space-between"
+   :text-align "center"})
+
+(defn TextField
+  [component-value]
+  [:> material-ui/TextField component-value])
+
+(defn Snackbar
+  [component-value]
+  [:> material-ui/Snackbar component-value])
+
+(defn Button
+  [component-value text]
+  [:> material-ui/Button component-value text])
+
+(defn call-save-api-fn
+  [title date]
+  (def request-map {:json-params {:title title :date date}})
```

Comment:
You should never use `def` within a `defn`. 
If you want a var which is to be used in a function you can use the `let` block to bind a name to an expression.

`def`s are used to create global variables. If you put it inside a `defn` the global var will be created every time the function is called. This will lead to problems when you have multiple requests (or threads) operating.